views/js/pciCreator/dev/audioRecordingInteraction/runtime/js/providers/mediaRecorder.js

```diff
@@ -45,8 +45,8 @@ define([
             chunkSizeMs = 100;      // size of a chunk (reduced from 1000ms to 100ms to avoid data loss in case of interrupted recording)
 
         var codecsByPreferenceOrder = [
-            'audio/webm;codecs=opus',
             'audio/ogg;codecs=opus',
+            'audio/webm;codecs=opus',
```

Comment:
Perhaps you can add another comment nearby to explain the reasons for the order.